tests/e2e/test_rosacli_upgrade.go

```diff
@@ -53,6 +53,14 @@ var _ = Describe("Cluster Upgrade testing",
 		})
 
 		AfterEach(func() {
+			if profile.Version == con.YStreamPreviousVersion {
+				By("Delete cluster upgrade")
+				output, err := upgradeService.DeleteUpgrade("-c", clusterID, "-y")
+				Expect(err).ToNot(HaveOccurred())
+				Expect(output.String()).To(ContainSubstring(
+					"Successfully canceled scheduled upgrade on cluster '%s'", clusterID))
```

Comment:
May I know more about the background of this fix? 
If this is a fix on some specific failed case, why cannot it delete the upgrade policy in the case block?

Reply:
@yuwang-RH this fix is for a CI job `rosa-sts-upgrade-y-stream` not for a specific test case,  here in test case `ocp-73731` an upgrade is scheduled but not deleted after running the case in AfterEach block, and in case `ocp-55883` it again tries to upgrade the same cluster and then due to the cluster has already an upgrade scheduled it fails to schedule another upgrade on that cluster.

more information on slack thread :- https://redhat-internal.slack.com/archives/C0519LZH1RA/p1724177182281089

Reply:
for the CI job `rosa-sts-upgrade-y-stream` more than one test case is doing upgrade task, so instead of having a `delete upgrade`  block for every case i guess better to keep it at AfterEach, please let me know if any better ideas we can use here. Thanks!!

Reply:
Why not trying to remove the upgrade all the time and not specific to specific Version (and forget about checking the error if no upgrade is there as it would fail in case upgrade is already deleted or ?)